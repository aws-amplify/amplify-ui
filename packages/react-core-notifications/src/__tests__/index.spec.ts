import * as exported from '..';

const sortedExportKeys = Object.keys(exported).sort();

describe('@aws-amplify/ui-react-core-notifications', () => {
  it('exports should match snapshot', () => {
    expect(sortedExportKeys).toMatchSnapshot();
  });
});
