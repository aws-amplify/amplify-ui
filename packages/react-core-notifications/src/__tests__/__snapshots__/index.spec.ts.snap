// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@aws-amplify/ui-react-core-notifications exports should match snapshot 1`] = `
Array [
  "InAppMessagingProvider",
  "handleMessageAction",
  "useInAppMessaging",
  "useMessage",
]
`;
